### ISSUE TITLE
(Possibly) uninitialized access to loop time

### ISSUE THREAD

**Issue by gyohng** [joyent/libuv#530]
Hello,

I found that the diagnostic tool that I'm using reports uninitialized access at this location (timer.c, windows branch ):

``` c
  if (ticks < time->LowPart) {
    time->HighPart += 1; // <<< here
  }
  time->LowPart = ticks;  // <<< and here
```

time seems to be derived from loop->time, and this part actually doesn't seem to be initialized in loop_init.

There are no side effects to this missing initialization that I discovered so far.

Thanks,
George.

**Comment by piscisaureus** [comment]
Thanks for reporting. Fixed in a7b83e0 and 2db0093.

**Comment by piscisaureus** [comment]
BTW, I would be very surprised if this:

```
time->LowPart = ticks; 
```

would actually be reported as uninitialized memory access...
